nbbuild/rat-exclusions.txt

```diff
@@ -175,6 +175,8 @@ websvccommon/websvc.saas.api/src/org/netbeans/modules/websvc/saas/oauth/*.templa
 **/*.ser
 harness/jellytools.platform/src/org/netbeans/jellytools/version_info
 **/*.npss
+### license breaks display in some usages
+nb/ide.branding/release/apache-netbeans.svg
```

Comment:
This is not necessary. There is an entry for `release/apache-netbeans.svg` in the `licenseinfo.xml` file and that in turn removes it from the rat auditing.

Reply:
Ah, I didn't realise it did that.  I checked with `ant rat`  before adding it in the license info.  Will remove from this file then.